lib/workers/repository/updates/branch-name.ts

```diff
@@ -110,7 +114,11 @@ export function generateBranchName(update: RenovateConfig): void {
     update.branchName = template.compile(update.branchName, update);
     update.branchName = template.compile(update.branchName, update);
   }
-
+  if (update.updateType === 'minor' && update.separateMultipleMinor) {
+    const newMinor = String(update.newMinor);
+    update.branchName = update.branchName.replace('.x', `.${newMinor}.x`);
+    logger.info(update.branchName);
```

Comment:
are we should about this info message? there should be at least some additional information

Reply:
Whoops no, let's remove